neo/Prompt/Commands/tests/test_config_commands.py

```diff
@@ -49,7 +49,7 @@ def test_config_sc_events(self):
         # test turning them off
         args = ['sc-events', '0']
         res = CommandConfig().execute(args)
-        self.assertTrue(res)
+        self.assertFalse(settings.log_smart_contract_events)

```

Comment:
Similarly to other config commands, I think this should still include the original `self.assertTrue(res)`. The additional change could be included also if desired.

Reply:
I've added this to the other commands as well (with the exception of NEP8)